lib/openregister.rb

```diff
@@ -215,7 +215,7 @@ def retrieve uri, type, base_url_or_phase, cache=nil, all=false, page_size=100
           items = Morph.from_tsv(tsv, type, OpenRegister)
           items.each do |item|
             additional_modification! item, base_url_or_phase, uri
-            unless item.key.nil?
+            unless item.instance_variable_defined?(:@key) && item.key.nil?
```

Comment:
This is getting the variable in item where the key is key defined by the instance variable `@key` if I understand correctly? Would it be good to have the second part of the statement `&& item[:key].nil?` to be consistent, or is this not equivalent?

Reply:
Looks like these are not equivalent as if I make this change I get a test failure with `NoMethodError: undefined method `[]' for #<OpenRegister::Country:0x007fc341ab91e8>`

Reply:
Understood.